jdaviz/app.py

```diff
@@ -972,6 +972,8 @@ def get_subsets(self, subset_name=None, spectral_only=False,
                 subset_region = self._get_range_subset_bounds(subset.subset_state,
                                                               simplify_spectral,
                                                               use_display_units)
+            elif isinstance(subset.subset_state, MultiMaskSubsetState):
+                subset_region = self._get_multi_mask_subset_definition(subset.subset_state)
             else:
```

Comment:
What else might still fall under this else?  Can it return any reasonable default in that case or raise an error instead?  I just worry about future cases where nothing getting returned could result in unintentionally missing something.

Reply:
Do you have a preference between reasonable default and raise error? I lean towards default but I can see the merits of either one.